### DISCUSSION TITLE
Auto find external materials

### DISCUSSION THREAD

**Discussion by natedawggh** [godotengine/godot-proposals#6484] [Editor]
When I bring assets into GODOT I'm often using already created materials saved to an external directory, but have to manually add each one to new assets over and over again. Some of the assets have 10-15 materials so this becomes extremely time consuming. It would save a lot of time if when clicking on "use external" in the import dialogue it automatically searched for the name of the applied material in the GODOT project and provided that link without having to navigate through the file directory each time. If there isn't an existing material with that name it could just stay empty.

**Comment by EsdrasCaleb** [comment]
Kenny one of the greater free assets providers is having this same issue
https://x.com/KenneyNL/status/1899857660985389482

he proposes to enable checking the use of external while selecting multiple assets. He also add that 3.x would automatic use an external material in all cases.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/7858.
